src/test/java/org/example/PersonTest.java

```diff
@@ -0,0 +1,171 @@
+package org.example;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class PersonTest {
+
+    @Test
+    void addParents() {
+        Person father = new Person("Jan", "Jong", 40,"male");
+        Person mother = new Person("Ana", "Jong", 40, "female");
+        Person person = new Person("Laura", "Jong", 20,"female");
+
+        person.addParents(mother, father);
+        Person result = person.getFather();
+
+        assertEquals(father, result);
+
+    }
+
+    @Test
+    void childShouldBeAddedToPerson() {
+        Person person = new Person("Laura", "Jong", 50,"female");
+        Person child1 = new Person("Ana", "Jong", 20, "female");
+        Person child2 = new Person("Jan", "Jong", 26,"male");
+
+        person.addChild(child1);
+        person.addChild(child2);
+        Person result = person.getChildren().get(1);
+
+        assertEquals(child2, result);
+
```

Comment:
Leuk dat je de opdracht met twee kinderen hebt uitgevoerd. In de uitwerkingen werken ze met List<Person> kids als vergelijking (waar jij result gebruikt): https://github.com/hogeschoolnovi/backend-java-family-tree-uitwerkingen/blob/main/src/test/java/PersonTest.java